public/v2/apps/mautic.json

```diff
@@ -0,0 +1,77 @@
+{
+  "captainVersion": "2",
+  "displayName": "Mautic beta",
+  "description": "Mautic is an open source marketing automation platform.",
+  "dockerCompose": {
+    "version": "2",
+    "services": {
+      "$$cap_appname-db": {
+        "documentation": "Taken from https://hub.docker.com/_/percona",
+        "image": "percona/percona-server:$$cap_percona_version",
+        "notExposeAsWebApp": "true",
+        "volumes": [
+          "$$cap_appname-db-data:/var/lib/mysql"
+        ],
+        "ports": [
+          "3306:3306"
+        ],
+        "restart": "always",
+        "environment": {
+          "MYSQL_ROOT_PASSWORD": "$$cap_db_pass",
+          "MYSQL_DATABASE": "mautic",
+          "MYSQL_USER": "$$cap_db_user",
+          "MYSQL_PASSWORD": "$$cap_db_pass"
+        }
+      },
+      "$$cap_appname": {
+        "depends_on": [
+          "$$cap_appname-db"
+        ],
+        "image": "mautic/mautic:$$cap_mautic_version",
+        "volumes": [
+          "$$cap_appname-data:/var/www/html"
+        ],
+        "restart": "always",
+        "environment": {
+          "MAUTIC_DB_HOST": "srv-captain--$$cap_appname-db",
+          "MYSQL_PORT_3306_TCP": "3306",
+          "MAUTIC_DB_USER": "$$cap_db_user",
+          "MAUTIC_DB_PASSWORD": "$$cap_db_pass",
+          "MAUTIC_DB_NAME": "mautic",
+          "MAUTIC_RUN_CRON_JOBS": "true"
+        }
+      }
+    }
+  },
+  "instructions": {
+    "start": "Mautic is an online,marketing automation tool.\nThe process will deploy a database and the App.\nThe process will take about a minute for the process to finish.",
+    "end": "Mautic is deployed and available as $$cap_appname .\nIf you dont have any SMTP server, I recommend you to install 'poste.io' one click app version."
+  },
+  "variables": [{
+    "id": "$$cap_db_user",
+    "label": "Database user",
+    "defaultValue": "mauticuser",
+    "validRegex": "/^([a-zA-Z0-9])+$/"
+  },
+    {
+      "id": "$$cap_db_pass",
+      "label": "Database password",
+      "description": "",
+      "validRegex": "/.{1,}/"
+    },
+    {
+      "id": "$$cap_mautic_version",
+      "label": "Mautic Version",
+      "defaultValue": "v3.0.0-beta",
```

Comment:
It appears that v3 is not a stable version yet, we should use v2.x, perhaps `2.13-apache`

Reply:
I think the 2.13 apache is even more buggy (last real production version [apache] has been updated 2 month ago, but last fixed version (excluding beta one) was not updated since 2 years)
I added beta into the name to warn about the beta version and do not want to use a not fixed version.

Reply:
information message is added on the description